modules/insights-operator-using-filters-on-advisor-recommendations.adoc

```diff
@@ -0,0 +1,32 @@
+// Module included in the following assemblies:
+//
+// * support/remote_health_monitoring/using-insights-to-identify-issues-with-your-cluster.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="insights-operator-using-filters-on-advisor-recommendations_{context}"]
+= Using filters to remove unnecessary results from Insights advisor recommendations
+
+As an OpenShift cluster manager, you can filter the recommendations that are displayed on the recommendations list. You might do this to reduce the number of reported recommendations or to concentrate on your highest priority issues. 
+
+Use the following example to select from several categories of recommendations to hide unneeded results.
```

Comment:
We can remove this sentence and try to make the below procedure generic, rather making it specific by applying only the Category filter.